lms/djangoapps/mobile_api/admin.py

```diff
@@ -3,7 +3,21 @@
 """
 
 from django.contrib import admin
-from config_models.admin import ConfigurationModelAdmin
-from mobile_api.models import MobileApiConfig
+from config_models.admin import ConfigurationModelAdmin, KeyedConfigurationModelAdmin
+from mobile_api.models import MobileApiConfig, AppVersionConfig
 
 admin.site.register(MobileApiConfig, ConfigurationModelAdmin)
+
+
+class AppVersionConfigAdmin(KeyedConfigurationModelAdmin):
+    """ Admin class for AppVersionConfig model """
+    fields = ('platform', 'version', 'expire_at', 'enabled')
+    list_filter = ['platform']
+
+    class Meta:  # pylint: disable=old-style-class
+        ordering = ['major_version', 'minor_version', 'patch_version']
```

Comment:
In the admin, it would make more sense to order versions in descending order so that the newer ones are at the top and so more reachable.